### PR TITLE
New translation of Latest Episode

### PR DESCRIPTION
**Pull request by steveculverwell**

### CODE REVIEW

File: src/RMP/Translate/lang/programmes/en.po [hunk 1]
```diff
@@ -1044,6 +1044,10 @@ msgstr "Last Played"
 msgid "latest"
 msgstr "Latest"
 
+#. English: "Latest Episode".
+msgid "latest_episode"
+msgstr "Latest Episode"
```
Comment (djmarland) [inline src/RMP/Translate/lang/programmes/en.po]: The standard (see the rest of the file) is only the first word is capitalised. So this should probably be
```Latest episode```

File: src/RMP/Translate/lang/programmes/en.po [hunk 2]
```diff
@@ -1044,6 +1044,10 @@ msgstr "Last Played"
 msgid "latest"
 msgstr "Latest"
 
+#. English: "Latest episode".
+msgid "latest_episode"
+msgstr "Latest Episode"
```
Comment (djmarland) [inline src/RMP/Translate/lang/programmes/en.po]: You missed it

File: src/RMP/Translate/lang/programmes/cy.po [hunk 3]
```diff
@@ -1049,6 +1049,10 @@ msgstr ""
 msgid "latest"
 msgstr "Diweddaraf"
 
+#. English: "Latest episode".
+msgid "latest_episode"
+msgstr "Pennod Ddiweddaraf"
```
Comment (djmarland) [inline src/RMP/Translate/lang/programmes/cy.po]: Also, this one should have a lower case D

File: src/RMP/Translate/lang/programmes/en.po (hunk 2 above)
Comment (steveculverwell) [inline src/RMP/Translate/lang/programmes/en.po]: I was going by designs. Ran it by a designer and they said yes it should be lower case.
Sorry.

File: src/RMP/Translate/lang/programmes/cy.po (hunk 3 above)
Comment (steveculverwell) [inline src/RMP/Translate/lang/programmes/cy.po]: I was going by designs. Ran it by a designer and they said yes it should be lower case.
Sorry.

**Comment by djmarland** [comment]
Yeah that happens a lot in designs. It's an easy mistake to make, so it's up to us to remind them of the standards already in place. Thanks for that.
👍

**Pull request merged**